SPECS/python-gevent/python-gevent.spec

```diff
@@ -40,6 +41,7 @@ Features include:
 
 %prep
 %autosetup -p 1 -n gevent-%{version}
+rm $(grep -rl '/\* Generated by Cython')
```

Comment:
Nit: could use a comment, so people know why we're doing this.

Reply:
Added, thanks